openmetadata-ui/src/main/resources/ui/src/components/Modals/IngestionRunDetailsModal/IngestionRunDetailsModal.tsx

```diff
@@ -0,0 +1,142 @@
+import ConnectionStepCard from '../../common/TestConnection/ConnectionStepCard/ConnectionStepCard';
+import { IngestionRunDetailsModalProps } from './IngestionRunDetailsModal.interface';
+
+function IngestionRunDetailsModal({
+  pipelineStatus,
+  handleCancel,
+}: Readonly<IngestionRunDetailsModalProps>) {
+  const { t } = useTranslation();
+  const [expandedKeys, setExpandedKeys] = useState<string[]>([]);
+
+  const columns: ColumnType<StepSummary>[] = useMemo(
+    () => [
+      {
+        title: t('label.step'),
+        dataIndex: 'name',
+        render: (_, record: StepSummary) => (
+          <Typography.Text data-testid={`step-summary-name-${record.name}`}>
+            {getEntityName(record)}
+          </Typography.Text>
+        ),
+      },
+      {
+        title: t('label.record-plural'),
+        dataIndex: 'records',
+      },
+      {
+        title: t('label.filtered'),
+        dataIndex: 'filtered',
+      },
+      {
+        title: t('label.warning-plural'),
+        dataIndex: 'warnings',
+      },
+      {
+        title: t('label.error-plural'),
+        dataIndex: 'errors',
+      },
+
+      {
+        title: t('label.failure-plural'),
+        dataIndex: 'failures',
+        render: (failures: StepSummary['failures'], record: StepSummary) =>
+          (failures?.length ?? 0) > 0 ? (
+            <Button
+              data-testid={`log-${record.name}`}
+              size="small"
+              type="link"
+              onClick={() => setExpandedKeys([record.name])}>
+              {t('label.log-plural')}
+            </Button>
+          ) : (
+            NO_DATA
+          ),
+      },
+    ],
+    [setExpandedKeys]
+  );
+
+  const expandable: ExpandableConfig<StepSummary> = useMemo(
+    () => ({
+      expandedRowRender: (record) => {
+        return (
+          record.failures?.map((failure) => (
+            <ConnectionStepCard
+              isTestingConnection={false}
+              key={failure.name}
+              testConnectionStep={{
+                name: failure.name,
+                mandatory: false,
+                description: failure.error,
+              }}
+              testConnectionStepResult={{
+                name: failure.name,
+                passed: false,
+                mandatory: false,
+                message: failure.error,
+                errorLog: failure.stackTrace,
+              }}
+            />
+          )) ?? []
+        );
+      },
+      indentSize: 0,
+      expandIcon: () => null,
+      expandedRowKeys: expandedKeys,
+      rowExpandable: (record) => (record.failures?.length ?? 0) > 0,
+    }),
+    [expandedKeys]
+  );
+
+  return (
+    <Modal
+      centered
+      destroyOnClose
+      open
+      closable={false}
+      maskClosable={false}
+      okButtonProps={{ style: { display: 'none' } }}
+      title={`Run status: ${startCase(
+        pipelineStatus?.pipelineState
+      )} at ${formatDateTime(pipelineStatus?.timestamp)}`}
```

Comment:
translation is missing

Reply:
Thanks. Nice catch.. Pushed the changes.